doc/ParkGuests.md

```diff
@@ -0,0 +1,61 @@
+How guests interact with the park defines gameplay, sets the challenges of the game and controls the game difficulty.
```

Comment:
looks a big design manual -ish, but I can also see it in the user manual.

It would be spiffy if we could have a state of implementation here, but I fear it's not feasible.